contrib/mesos/docs/ha.md

```diff
@@ -66,5 +86,417 @@ The command line parameters that affect the hash calculation are listed below.
 - `--proxy-*`
 - `--static-pods-config`
 
+## Cold-standby
+
+Setting up Kubernetes on Mesos in cold-standby mode is similar to Kubernetes in
+standalone mode described in [Kubernetes HA][1]. However, special attention is
```

Comment:
I'm not comparing cold vs. hot here. I was comparing K8sm vs K8s when setting up HA. If you set up HA in K8s, you don't have to worry about different schedulers might have different executor IDs. Whereas for K8sm schedulers, you need to specify its parameters consistently across so they all generate the same executor ID.